packages/tfm-python38/tfm-python38.spec

```diff
@@ -0,0 +1,144 @@
+%global __python /usr/bin/python3
+%global scl_name_base tfm-python
+%global scl_name_version 38
+%global scl %{scl_name_base}%{scl_name_version}
+
+%scl_package %scl
+
+%global scl_python rh-python38
+%global scl_prefix_python %{scl_python}-
+
+%global install_scl 1
+
+# Do not produce empty debuginfo package.
+%global debug_package %{nil}
+
+Summary: Package that installs %scl
+Name: %scl_name
+Version: 1.0
+Release: 1%{?dist}
+License: GPLv2+
+Source1: README
+Source2: LICENSE
+BuildRequires: help2man
+BuildRequires: scl-utils-build
+BuildRequires: %{scl_prefix_python}scldevel
+BuildRequires: %{scl_prefix_python}python-devel
+%if 0%{?install_scl}
+Requires: %{scl_python}
+%endif
+
+%description
+This is the main package for %scl Software Collection.
+
+%package runtime
+Summary: Package that handles %scl Software Collection.
+Requires: scl-utils
+Requires: %{scl_prefix_python}runtime
+
+%description runtime
+Package shipping essential scripts to work with %scl Software Collection.
+
+%package build
+Summary: Package shipping basic build configuration
+Requires: scl-utils-build
+Requires: %{scl_runtime}
+Requires: %{scl_prefix_python}scldevel
+
+%description build
+Package shipping essential configuration macros to build %scl Software Collection.
+
+%package scldevel
+Summary: Package shipping development files for %scl
+
+%description scldevel
+Package shipping development files, especially useful for development of
+packages depending on %scl Software Collection.
+
+%package python3-devel
+Summary: Package pulling in %{scl_prefix_python}python-devel
```

Comment:
Maybe something like `empty package to pull in....` or include somewhere in the description a note about *why* we have to do this? Thinking of things to help us remember or inform any othres that come looking at this code.